### PR TITLE
Implemented counter to favourite beers

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 1]
```diff
@@ -31,20 +32,26 @@ class FavouriteBeersComponent extends PureComponent {
   };
 
   deleteBeer = (id) => {
+    const clicked = true;
+    this.setState({ clicked });
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: No need to create new variable. I'd create new method for that:
```
toggleDeleteButtonClicked = () => {
  this.setState(prevState => ({
    deleteButtonClicked: !prevState.deleteButtonClicked
   }));
 };
```

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 2]
```diff
@@ -61,7 +68,9 @@ class FavouriteBeersComponent extends PureComponent {
                 className={beerIdToDelete === id ? 'opacity-anim' : ''}
                 key={id}
               >
-                <DeleteButton onClick={() => this.deleteBeer(id)} />
+                {clicked === false ? (
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: I'd go with:
`!clicked && <DeleteButton ...`

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 3]
```diff
@@ -354,7 +354,7 @@ function SearchBeersComponent() {
         type="button"
         onClick={clearFilter}
         className="button-clean-filter"
-        text="clean the filter"
+        text="Clean the filter"
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: Clear

**Review by mi-chal — APPROVED** [review]

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 1 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Done

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 2 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Fixed

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 3 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Fixed

**Pull request merged**